src/main/java/com/micetr0/controller/CompositionController.java

```diff
@@ -7,8 +7,51 @@
 
 package com.micetr0.controller;
 
+import com.micetr0.model.Composition;
+import com.micetr0.model.Section;
+
+import java.util.List;
+
 public class CompositionController {
 
-    //TODO: Implement
+    Composition composition = new Composition();
+
+    public void setModel(Composition composition) {
+        this.composition = composition;
+    }
+
+    public boolean deleteSection(int index)
+    {
+      List<Section> sections = composition.getSections();
+
+      if(sections.size() < 1 || (sections.size() < index+1))
+      {
+          // don't do anything, theres nothing to remove
+          return false;
+      }
+      else{
+          sections.remove(index);
+          composition.setSections(sections);
+          return true;
+      }
+    }
+    public boolean addSection()
```

Comment:
Composition controller should call the Section controller to add new sections, and section controller should return a section.